### PR TITLE
Various minor fixes.

### PR DESCRIPTION
**Pull request by averms**
Changed some lines to use semantic linefeeds, fixed a few typos, fixed
some odd English.

### CODE REVIEW

File: encoding/preparation.md [hunk 1]
```diff
@@ -296,4 +296,6 @@ Just extract it somewhere.
 
 ---
 
-[^1]: It should be noted that the author strongly disagrees with this sentiment. The two have a lot in common, and any capable Avisynth encoder could reach a similar leven in Vapoursynth within a few months, maybe even weeks. At least I'm honest, okay?
```
Comment (OrangeChannel) [inline encoding/preparation.md:299]: Gitbook doesn't like line breaks in footnotes. It breaks the auto footnote formatting. Other than that, 👍

File: encoding/preparation.md [hunk 2]
```diff
@@ -296,4 +296,6 @@ Just extract it somewhere.
...
+[^1]: It should be noted that the author strongly disagrees with this sentiment.
```
Comment (FichteFoll) [inline encoding/preparation.md]: As noted in https://github.com/Irrational-Encoding-Wizardry/guide.encode.moe/commit/a0280ae57b29b1db294a51c06f4eec6d2b394c42, semantic linefeeds don't work for footnotes. It looks like we should add this to `CONTRIBUTING.md`.

File: encoding/masking-limiting-etc.md [hunk 3]
```diff
@@ -294,7 +294,7 @@ out = core.std.FrameEval(src, partial(scsmooth, clip=src, ref=ref), prop_src=pro
 
 TODO
 
-Example: Deband a grainy clip with f3kdb (16 bit input)
+Example: deband a grainy clip with f3kdb (16 bit input)
```
Comment (FichteFoll) [inline encoding/masking-limiting-etc.md]: I'm no expert in English punctuation, but in German you would want to start with a capital letter after a colon when the following section is a full sentence by itself. Maybe you could clarify or provide a reference for this? (I'm interested in this personally.)

**Review by FichteFoll — CHANGES_REQUESTED** [review]
Thank you for your contribution.

Please see my comments below, notably the footnotes one.

File: encoding/masking-limiting-etc.md (hunk 3 above)
Comment (averms) [reply encoding/masking-limiting-etc.md]: Well English is super ambiguous about these kinds of things. The Chicago manual of style agrees with me but the APA and MLA seem to agree with you.
Looking at it again, it does make more sense to capitalize as this isn't exactly a complete sentence.

**Comment by averms** [comment]
Should be all good now.

**Review by OrangeChannel — APPROVED** [review]

File: encoding/preparation.md (hunk 1 above)
Comment (averms) [reply encoding/preparation.md:299]: Ok but I didn't do that, I only changed the spelling: leven -> level

Comment (OrangeChannel) [reply encoding/preparation.md:299]: Oh, I just clicked approve, not sure why it posted this comment again... This was from 2 days ago on the initial PR, sorry about that.

**Pull request merged**